ec2api/api/route_table.py

```diff
@@ -480,18 +480,26 @@ def _update_routes_in_associated_subnets(context, route_table, cleaner,
 
 def _update_subnet_host_routes(context, subnet, route_table, cleaner=None,
                                rollback_route_table_object=None,
-                               router_objects=None, neutron=None):
+                               router_objects=None, neutron=None,
+                               vpc_route_revert=False, add_vpc_route=False):
```

Comment:
Yea. Sure. This was in place coz of the earlier idea that we will prevent vpc routes no matter what.  It wont matter anyways since customer wont be able to add vpc route or and specific route for that matter. But I will revert the same